web/ui/src/app.js

```diff
@@ -20,7 +20,7 @@ var DEBUG = false;
 /*******************************************************************************
  * Main module & controllers
  ******************************************************************************/
-var controlplane = angular.module('controlplane', ['ngRoute', 'ngCookies','ngDragDrop','pascalprecht.translate', 'angularMoment', 'zenNotify', 'serviceHealth', 'ui.datetimepicker', 'modalService', 'angular-data.DSCacheFactory', 'ui.codemirror', 'sticky', 'graphPanel', 'servicesFactory', 'healthIcon', 'authService', 'miscUtils', 'hostsFactory']);
+var controlplane = angular.module('controlplane', ['ngRoute', 'ngCookies','ngDragDrop','pascalprecht.translate', 'angularMoment', 'zenNotify', 'serviceHealth', 'ui.datetimepicker', 'modalService', 'angular-data.DSCacheFactory', 'ui.codemirror', 'sticky', 'graphPanel', 'servicesFactory', 'healthIcon', 'authService', 'miscUtils', 'hostsFactory', 'poolsFactory']);
```

Comment:
Readability nit - should the array of dependencies be broken out into one dependency/line?